kolibri/core/assets/src/views/ExamReport/PageStatus.vue

```diff
@@ -12,40 +12,20 @@
         </h1>
         <KLabeledIcon icon="quiz" :label="contentName" />
       </div>
-
-      <table class="scores">
-        <tr>
-          <th>
-            {{ $tr('overallScore') }}
-          </th>
-          <td>
-            <strong>
-              {{ $formatNumber(score, { style: 'percent' }) }}
-            </strong>
-          </td>
-        </tr>
-        <tr>
-          <th>
-            {{ $tr('questionsCorrectLabel') }}
-          </th>
-          <td>
-            {{ $tr('questionsCorrectValue', {
-              correct: questionsCorrect, total: questions.length
-            }) }}
-          </td>
-        </tr>
-      </table>
+      <!-- only show the current try if the user has only one try -->
+      <CurrentTryOverview
+        :progress="progress"
+        :questionsCorrect="questionsCorrect"
+        :score="score"
+        :totalQuestions="questions.length"
+        :completionTimestamp="completionTimestamp"
+        :completed="completed"
+      />
     </KFixedGridItem>
     <KFixedGridItem span="1" alignment="right">
-      <div>
-        <ProgressIcon class="svg-icon" :progress="progress" />
-        <strong>
-          {{ progressIconLabel }}
-        </strong>
-      </div>
-      <div v-if="completed">
-        <ElapsedTime :date="completionTimestamp" />
-      </div>
+      <KButton v-if="retry" @click="$emit('repeat')">
+        {{ coreString('practiceAgainButton') }}
```

Comment:
There is a slight change here - I [updated it to be called the `tryAgainButton`](https://github.com/learningequality/kolibri/pull/8682) in the most recent string update in `develop` [based on this conversation](https://github.com/learningequality/kolibri/pull/8529/files#r747727942). No more changes to the code after this! Sorry about that on my end.

Reply:
Updated!